process/block/metrics.go

```diff
@@ -125,8 +125,8 @@ func incrementCountAcceptedBlocks(
 
 	bitMap := header.GetPubKeysBitmap()
 
-	indexInBitmap := bitMap[myIndex/8]&(1<<uint8(myIndex%8)) == 1
```

Comment:
could have worked also if the condition would have been `> 0` :)

Reply:
done.. with `!= 0`